std_msgs/Float64MultiArray features
KeyPointMap[] keypointmaps
